### PR TITLE
typo in virtual property

### PR DESCRIPTION
**Pull request by Xbirdfr**

### CODE REVIEW

File: fr/orm/entities.rst [hunk 1]
```diff
@@ -205,8 +205,8 @@ nom complet::
 
         protected function _getFullName()
         {
-            return $this->_properties['first_name'] . '  ' .
-                $this->_properties['last_name'];
+            return $this->_fields['first_name'] . '  ' .
+                $this->_fields['last_name'];
```
Comment (garas) [inline fr/orm/entities.rst]: It is better `$this->first_name . '  ' . $this->last_name` like in English.

Same issue in pt + ja.

Comment (saeideng) [reply fr/orm/entities.rst]: yes using `->first_name` calls `get()` and it is safe to use :+1:

Comment (garas) [reply fr/orm/entities.rst]: Also properties are usually annotated with `@property`, which helps further for IDE and static analysis.

Comment (markstory) [reply fr/orm/entities.rst]: Merging as these suggestions are great but the proposed changes are still better than before.

**Pull request merged**